src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -103,6 +103,10 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
                 continue;
             }
 
+            if (!$showHidden && $builder->hasDefinition($serviceId) && $builder->getDefinition($serviceId)->hasTag('container.excluded')) {
```

Comment:
This won’t work if the service is an alias (see hasDefinition() vs has())

Reply:
Thanks. I’m fixed with help use existing variable `$service`. Do you have any other comments?